docs/cristin.ttl

```diff
@@ -1,57 +1,299 @@
-    rdf:type     rdf:Property ;
-    rdfs:label   "The secondary category of the resource" ;
+:Result
+    a            rdfs:Class ;
+    rdfs:comment "The equivalent of an NVA Resource in Cristin. An entry as being exported especially for importing in NVA" .
+
+:PublicationStatus
+    a rdfs:Class .
+
+:Published
+    a               rdfs:Class ;
+    rdfs:label      "PUBLISHED" ;
+    rdfs:subClassOf :PublicationStatus .
+
+:VARBEID_SPRAK_Class
+    a            rdfs:Class ;
+    rdfs:comment "The structure defining the VARBEID_SPRAK property" .
+
+:VARBEID_PERSON_Class
+    a            rdfs:Class ;
+    rdfs:comment """
+    The structure of the VARARBEID_PERSON entry. It contains information about the status of the contributor
+    at the time the Result was registered in Cristin.
+    (i.e., not necessarily the time the Result was registered in Cristin)
+    """ .
+
+:VARBEID_PERSON_STED_Class
+    a            rdfs:Class ;
+    rdfs:comment """The structure of the VARBEID_PERSON_STED entry.
+    It contains information about the status of a contributor's institution at the time the Result
+    was registered in Cristin. It is uncertain whether the contributor presently belongs to the same institution,
+    or if the institution status has changed.
+    """ .
+
+:VARBEID_PERSON_STED_ROLLE_Class
+    a            rdfs:Class ;
+    rdfs:comment """ The structure of VARBEID_PERSON_STED_ROLLE. It contains information about the role of
+    the contributor in their institution at the time the Result was registered in Cristin. It is uncertain
+    whether the contributor presnetly holds the same role, or they belong to the same institution, or the
+    role still exists, or the institution's status has changed.
+""" .
+
+:id
+    a            rdf:Property ;
+    rdfs:comment "Identifier of Cristin Result entry" ;
+    rdfs:range   xsd:positiveInteger ;
+    rdfs:domain  :Result .
+
+:arstall
+    a            rdf:Property ;
+    rdfs:comment "The year a resource (result) was published" ;
+    rdfs:range   xsd:positiveInteger ;
+    rdfs:domain  :Result .
+
+:dato_opprettet
+    a            rdf:Property ;
+    rdfs:comment "The date the resource was registered in Cristin" ;
+    rdfs:range   xsd:date ;
+    rdfs:domain  :Result .
+
+:varbeidhovedkatkode
+    a            rdf:Property ;
+    rdfs:comment """
+    The main category of the resource,
+    The range of this property is found here: https://api.cristin.no/v2/results/categories?lang=nb,en
+    """ ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :Result .
+
+:varbeidunderkatkode
+    a            rdf:Property ;
+    rdfs:comment "The secondary category of the resource" ;
     rdfs:comment "No specific range available at the moment" ;
     rdfs:range   xsd:string ;
-    rdfs:domain  cr:Result .
+    rdfs:domain  :Result .
+
+:eierkode_opprettet
+    a            rdf:Property ;
+    rdfs:comment "The institution of the person who created the first version of the registration " ;
```

Comment:
To make things look a bit tidier, remove the leading and trailing spaces in the comments throughout the document.